docs/book/src/dist/dist.md

```diff
@@ -96,17 +96,17 @@ Now generate a token for the server, giving the IP and port the scheduler and cl
 connect to the server on (address `192.168.1.10:10501` here):
 
 ```sh
-cachepot-dist auth generate-jwt-hs256-server-token \
+cachepot-dist auth generate-jwt-hs256-coordinator-token \
     --secret-key YOUR_KEY_HERE \
-    --server 192.168.1.10:10501
+    --coordinator 192.168.1.10:10501
 ```
 
 *or:*
 
 ```sh
-cachepot-dist auth generate-jwt-hs256-server-token \
+cachepot-dist auth generate-jwt-hs256-coordinator-token \
     --config /path/to/scheduler-config.toml \
-    --server 192.168.1.10:10501
+    --coordinator 192.168.1.10:10501
```

Comment:
I think this is still incorrect.

Reply:
This refers to dist-servers rather than coordinators, right?

Reply:
Yep